### DISCUSSION TITLE
Cannot login after fresh Horus install.

### DISCUSSION THREAD

**Discussion by iamoat** [elementary/triage#227] [Q&A] [answered]
I cannot login to the OS. Got a message "Account Disabled" under my Login Avatar. 

I did try reinstall but won't work. 

This is my first time install Elementary.

**Comment by tintou** [comment]
Hi there, this shouldn't be the case, can you provide us the username that you wrote and the host name?

**Comment by amarraff** [comment]
Sorry to bump this after so long, but it's the only discussion I see regarding the issue - as of Elementary 8.1 a quick reboot upon reaching this bug fixed it for me.

**Comment by ryonakano** [comment, marked as answer]
Duplicate of https://github.com/elementary/initial-setup/issues/222 and fixed by https://github.com/elementary/greeter/pull/879 which is not yet released.
